modules/ossm-migrating-network-policies-security.adoc

```diff
@@ -0,0 +1,199 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/checklists/ossm-migrating-network-policies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-migrating-network-policies-security_{context}"]
+= Migrating network policies for security reasons
+
+This procedure applies only if you must recreate your network policies first for security reasons.
+
+.Prerequisites
+
+* You have deployed {ocp-product-title} 4.14 or later.
+* You are logged in to the {ocp-product-title} web console as a user with the cluster-admin role.
+
+[IMPORTANT]
+====
+The `maistra.io/member-of: istio-system` label will be removed from the namespaces during migration.
```

Comment:
nit: this will have different values depending on what namespace the SMCP is in so you can just use the "key".
```suggestion
The `maistra.io/member-of` label will be removed from the namespaces during migration.
```